Source/swiftlint/Helpers/LintOrAnalyzeArguments.swift

```diff
@@ -0,0 +1,41 @@
+import ArgumentParser
+
+// MARK: - Common Arguments
+
+struct LintOrAnalyzeArguments: ParsableArguments {
+    @Option(help: "The path to one or more SwiftLint configuration files, evaluated as a parent-child hierarchy.")
+    var config = [String]()
+    @Flag(name: [.long, .customLong("autocorrect")], help: "Correct violations whenever possible.")
+    var fix = false
+    @Flag(help: "Use an alternative algorithm to exclude paths for `excluded`, which may be faster in some cases.")
+    var useAlternativeExcluding = false
+    @Flag(help: "Read SCRIPT_INPUT_FILE* environment variables as files.")
+    var useScriptInputFiles = false
```

Comment:
I wonder if it's worth trying to make any of these mutually exclusive in the future

Reply:
Yes that'd be nice. Nothing stopping me from doing this now either. There's also `--reporter` that I'd like to make an enum instead of a string.

Reply:
Unfortunately, I can't make `useScriptInputFiles` mutually exclusive with options outside this option group.

This is nice though:

```
swiftlint --strict --lenient
Error: Value to be set with flag '--lenient' had already been set with flag '--strict'
Usage: swiftlint lint <options>
  See 'swiftlint lint --help' for more information.
```

Reply:
It doesn't look like using an enum for `--reporter` gets me anything, but it makes the error reporting worse and adds more cruft, so I'm keeping it a string for now.